selfdrive/car/honda/hondacan.py

```diff
@@ -167,7 +167,8 @@ def create_ui_commands(packer, CAN, CP, enabled, pcm_speed, hud, is_metric, acc_
     commands.append(packer.make_can_msg("ACC_HUD", CAN.pt, acc_hud_values))
 
   lkas_hud_values = {
-    'SET_ME_X41': 0x41,
+    'ENABLED': 1,
```

Comment:
Can you rename enabled to something more clear? We have LDW_ON/OFF in the same message

Reply:
Just enabled. Available might be a good alternative.